ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -0,0 +1,6690 @@
+periodics:
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 4h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-release: "4.7"
+  name: promote-release-openshift-machine-os-content-e2e-aws-4.7
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --kubeconfig=/etc/apici/kubeconfig
+      - --lease-server-password-file=/etc/boskos/password
+      - --lease-server-username=ci
+      - --lease-server=https://boskos-ci.svc.ci.openshift.org
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-aws-cluster-profile
+      - --template=/usr/local/e2e-aws
+      - --input-hash=$(BUILD_ID) --input-hash=$(JOB_NAME)
+      - --promote
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        # prow doesn't allow init containers or a second container
+        export PATH=$PATH:/tmp/bin
+        mkdir /tmp/bin
+        curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.6/linux/oc.tar.gz | tar xvzf - -C /tmp/bin/ oc
```

Comment:
something is wrong here `clients/oc/4.6/linux`

Reply:
hm... yea that link doesn't exist. Shall we use 4.5 ?